designs/images/cluster_reconcile.puml

```diff
@@ -1,4 +1,4 @@
-@startuml
+@startuml cluster_reconcile
```

Comment:
What does this show to create the above png diff?

Reply:
This defined the name of the image file that plantuml generates (`cluster_reconcile.png`)
It should be required for all diagrams, this one was missing it